content/en/blog/_posts/2023-08-22-sig-contribex-spotlight.md

```diff
@@ -0,0 +1,294 @@
+---
+layout: blog
+title: "Spotlight on SIG ContribEx"
+date: 2023-08-22
```

Comment:
```suggestion
date: 2023-08-14
```
If we miss the 14th, I recommend waiting until after the post-release articles for K8s 1.28 are done (takes a couple of week). It's weird to mix 'em up.

Release comms gatekeep the main blog during that post-release phase.

Reply:
Sure, I'm up for it and I think it's doable, I'll be on top of this today and tomorrow. And yes, better get it our before the release.